src/app/Main/Body/QueryPage/QueryPageContent-Layout.component.tsx

```diff
@@ -15,22 +15,15 @@ import { QueryPageContentBarChartResultsComponent } from "./Results/Charts/Query
 const contentStyle = css`
     width: 0;

```

Comment:
see flex

Reply:
I don't understand this comment.

Reply:
I think you used ``width: 0`` for the initial width so the content has the lowest flex priority and only can consume the remaining space in the flex row. So it also belongs to the QueryPage-Layout.